### DISCUSSION TITLE
Restructure the structure

### DISCUSSION THREAD

**Discussion by paskino** [TomographicImaging/CIL-Demos#58] [General]
Rename the the directory `examples` to `tutorials`. 

Use the `examples` to host some simpler snippets of code referring to structures in CIL. 

We expect to have a structure following `cil`'s with `io`, `optimisation` etc etc

**Comment by paskino** [comment]
These should also be tested automatically with some GHA or similar

**Comment by lauramurgatroyd** [comment]
CLosing this as we now have the demos folder
